index.html

```diff
@@ -6030,6 +6035,435 @@ <h2>Required</h2>
+                    },
+                    "adjustRpm": {
+                        "type": "number",
+                        "minimum": 200,
+                        "maximum": 1200
+                    }
+                }
+            }
+           </pre>
+           <pre class="LED exampleTab1">
+            {
+                "@context": "http://www.w3.org/ns/td",
+                "@type": "tm:ThingModel",
+                "title": "LED Thing Model",
+                "version": {
+                    "model": "1.0.0"
+                },
+                "properties": {
+                    "R": {
+                        "type": "number",
+                        "description": "Red color"
+                    },
+                    "G": {
+                        "type": "number",
+                        "description": "Green color"
+                    },
+                    "B": {
+                        "type": "number",
+                        "description": "Blue color"
+                    }
+                },
+                "actions": {
+                    "fadeIn": {
+                        "title": "fadeIn",
+                        "input": {
+                            "type": "number",
+                            "description": "fadeIn in ms"
+                        }
+                    },
+                    "fadeOut": {
+                        "title": "fadeIn",
+                        "input": {
+                            "type": "number",
+                            "description": "fadeOut in ms"
+                        }
+                    }
+                }
+            }
+           </pre>
+         </aside>
+
+
+    <p>
+        Such composition can be introduced by the usage of the <code>links</code> container: 
+        <span class="rfc2119-assertion" id="tm-compose-submodel">
+            A parent <a>Thing Model</a> MUST use at least one <code>links</code> entry with <code>"rel":"tm:submodel"</code> 
+            that targets to a (sub-) or child <a>Thing Model</a> that is be composed.
+        </span>
+        <span class="rfc2119-assertion" id="tm-compose-instanceName">
+            Optional an <code>instanceName</code> can be provided to associate an individual name to the composed (sub-) <a>Thing Model</a>.
+        </span> 
+        This is useful when multiple same <a>Thing Model</a> definitions are composed and needs to be distinguised. 
+    </p>
+
+
+    <p>
+        Different strategies can be followed to generate <a>Thing Descriptions</a> from composed <a>Thing Model</a> definitions. The default recommendation is to generate from each
+         parent and sub/child <a>Thing Model</a> a corresponding <a>Thing Descriptions</a> (also see <a href="#thing-model-td-generation"></a>). 
+       The composition relation can be reflected by the <code>collection</code> and <code>item</code> relation types in the links container of the 
```

Comment:
is this really a "The composition relation `can` be reflected"? our should this be `SHOULD`?

Reply:
Here I would keep it optional if Thing providers want to share what the relationship looks like with other Things during run time. This may also be avoided for security reasons.